README.md

```diff
@@ -1,57 +1,149 @@
+create an element to render our title in (let's call it a container). Remember how we added a text child in our `title`
+component? Now we're going to add an element as a child, so we pass it by reference instead of using a string:
+
+```js
+const title = React.createElement('h1', {}, 'My First React Code');
+const container = React.createElement('div', {}, title);
+
+ReactDOM.render(
+  container,
+  document.getElementById('main')
+);
+```
+
+Our `title` seems a little lonely though... Let's add a sibling! We'll create a `paragraph` element and then pass it as
+another child to our `container` children.
+
+```js
+const title = React.createElement('h1', {}, 'My First React Code');
+const paragraph = React.createElement('p', {}, 'Writing some more HTML. Cool stuff!');
+const container = React.createElement('div', {}, [title, paragraph]);
 
-```javascript
-ReactDOM.render(heading, document.getElementById('main'))
+ReactDOM.render(
+  container,
+  document.getElementById('main')
+);
 ```
 
-Now add children:
+Note how if we want to add multiple children, we use an _array_!
 
-```javascript
+## Your children's children
+We can nest children as much as we want. We also don't need to store our elements in variables before using them, we can
+declare them inline as well (though the downside of this is less readable code):
+
+```js
+const list =
+  React.createElement('div', {},
+    React.createElement('h1', {}, 'My favorite ice cream flavors'),
+    React.createElement('ul', {},
+      [
+        React.createElement('li', {}, 'Chocolate'),
+        React.createElement('li', {}, 'Vanilla'),
+        React.createElement('li', {}, 'Banana')
+      ]
+    ));
+
+ReactDOM.render(list, document.getElementById('main'));
+```
+
+## Adding attributes
+As mentioned before, we pass properties to an element using the second argument (an object, which we've left empty for
+now). Suppose we wanted to add some classes to make our ice cream flavors stand out. What would we need to add? Let's
+try this...
+
+```js
 const list =
   React.createElement('div', {},
-    React.createElement('h1', {}, 'People'),
+    React.createElement('h1', {}, 'My favorite ice cream flavors'),
     React.createElement('ul', {},
-      React.createElement('li', {}, 'Someone Cool'),
-      // etc.
-      ))
+      [
+        React.createElement('li', {}, 'Chocolate'),
+        React.createElement('li', {}, 'Vanilla'),
+        React.createElement('li', {}, 'Banana')
+      ]
+    ));
+
+ReactDOM.render(list, document.getElementById('main'));
+```
+
+Oops! That doesn't seem to work. If we take a look at our console, we'll see a helpful error message (ignore the error
+message about a `key` property, we'll get to that in another lesson):
 
-ReactDOM.render(list, document.getElementById('main'))
+```
+Warning: Unknown DOM property class. Did you mean className?
+    in li
+    in ul
+    in div
```

Comment:
Great that you explain the errors that are likely to confused. Allows for a nice foreshadowing too.